HW2_Bredov/calculator.py

```diff
@@ -0,0 +1,25 @@
+def divide(a, b):
+  return a/b
+
+def add(a,b):
+    return a+b
+
+def multiply(x,y):
+    return(x*y)
+  
+def subtract(eq1,eq2):
+	return(eq1-eq2)
```

Comment:
```suggestion
def subtract(num1, num2):
	return num1 - num2
```

Что значит eq? 😄 Мы же не уравнения вычитаем